substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -193,6 +192,13 @@ pub mod pallet {
 		#[pallet::constant]
 		type BondingDuration: Get<EraIndex>;
 
+		/// Maximum number of eras that staked funds could remain bonded for.
+		///
+		/// Ideally, it should be `BondingDuration::get() + 1` to include all eras in the interval
+		/// [current_era - BondingDuration::get(), current_era].
+		#[pallet::constant]
+		type MaxBondedEras: Get<u32>;
```

Comment:
we should have an integrity test ensuring `MaxBondedEras` >= `BondingDuration + 1`

Reply:
Added in tests

```
assert_eq!(MaxBondedEras::get(), (BondingDuration::get() as u32) + 1);
```

Reply:
`integrity_test` is used in a test at the runtime declaration. So it tests for runtimes, instead of just the mock